lib/sentry.ex

```diff
@@ -335,7 +335,7 @@ defmodule Sentry do
         Client.send_event(event, opts)
 
       !Config.dsn() ->
-        :ignored
+        Client.validate_and_ignore(opts)
```

Comment:
This is gonna make it harder to track the return value–I have to jump to another module. Maybe we can do

```elixir
Client.validate_options!(opts)
:ignored
```

instead?

Reply:
Yes, good idea!